app/javascript/packs/validators/components/validator_score_modal.vue

```diff
@@ -70,22 +70,20 @@
                 </td>
               </tr>
 
-              <!--
               <tr>
-                <td class="column-md fw-bold">Skipped After Score</td>
+                <td class="column-md fw-bold">Vote Latency Score</td>
                 <td class="column-xs text-nowrap">
-                  <i :class="'score-' + parseInt(score.skipped_after_score)"
+                  <i :class="'score-' + parseInt(score.vote_latency_score)"
                      class="fa-solid fa-circle me-1">
                   </i>
-                  {{ score.skipped_after_score }}
+                  {{ score.skipped_slot_score }}
```

Comment:
It should show slot latency score, not skipped_slot_score twice.

Also, it still shows "max 11" on this modal instead of 13.